<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutWidget</class>
 <widget class="QWidget" name="AboutWidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="mainLabel">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextEditable</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="LicenceTextBrowser"/>
   </item>
   <item>
    <widget class="QLabel" name="teamLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="teamTextBrowser"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
